lib/xcodeproj/scheme/archive_action.rb

```diff
@@ -0,0 +1,40 @@
+module Xcodeproj
+  class XCScheme
+    class ArchiveAction < XMLElementWrapper
+      def initialize(node = nil)
+        create_xml_element_with_fallback(node, 'ArchiveAction') do
+          self.build_configuration = 'Release'
+          self.reveal_archive_in_organizer = true
+        end
+      end
+
+      def build_configuration
+        @xml_element.attributes['buildConfiguration']
+      end
```

Comment:
Yeah I thought about a Mix-in but then I was afraid that it was a bit overkill (as this can be applied to other methods that are use in just 2 classes, etc). Will consider it for buildConfiguration anyway as this is used in more than 2 classes!